epsagon/handler.py

```diff
@@ -0,0 +1,28 @@
+"""
+Epsagon generic wrapper used only in Lambda environments.
+"""
+
+from .utils import init, import_original_module
+from .wrappers import lambda_wrapper
+
+init()
+ORIGINAL_MODULE = import_original_module()
+
+
+def wrapper(event, context):
+    """
+    Generic wrapper for Lambda functions.
+    :param event: Lambda's event
+    :param context: Lambda's context
+    :return: Original handler
+    """
+    original_module, module_path, handler_name = ORIGINAL_MODULE
+
+    try:
+        wrapped_handler = getattr(original_module, handler_name)
+    except AttributeError:
+        raise AttributeError(
+            'No handler {} in module {}'.format(handler_name, module_path)
+        )
+
+    return lambda_wrapper(wrapped_handler)(event, context)
```

Comment:
it's not good that we have to evaluate the wrapper for each execution. let's do it at init

Reply:
Done.